test/support/builders/metadata/base.js

```diff
@@ -105,12 +111,12 @@ module.exports = class BaseMetadataBuilder {
   }
 
   newerThan (doc /*: Metadata */) /*: this */ {
-    this.doc.updated_at = new Date(timestamp.fromDate(doc.updated_at) + 2000)
+    this.doc.updated_at = new Date(timestamp.fromDate(doc.updated_at).getTime() + 2000)
     return this
   }
 
   olderThan (doc /*: Metadata */) /*: this */ {
-    this.doc.updated_at = new Date(timestamp.fromDate(doc.updated_at) - 2000)
+    this.doc.updated_at = new Date(timestamp.fromDate(doc.updated_at).getTime() - 2000)
```

Comment:
We had to fix metadata builder's newerThan / olderThan for our unit test to actually show different update_at values.